backend/src/email/interfaces/email.interface.ts

```diff
@@ -41,7 +41,7 @@ export interface EmailDuplicateCampaignDetails
  *          - status
  *        properties:
  *          id:
- *            type: number
+ *            type: string
```

Comment:
just thinking out loud — I read that in API design, we should "be liberal in what you accept, be conservative in what you send". do you think it's a good idea for us to be able to accept both and convert accordingly?

Reply:
Not sure why u're commenting on this irrelevant line

Reply:
Moving it to where it's relevant